app/gcl/agwa/section2/CollapsibleSection.tsx

```
@@ -0,0 +1,83 @@
+
+    const toggle = () => setOpen((prev) => !prev);
+
+    return (
+        <div className={styles['section-card']} id={id} style={wrapperStyle}>
+            <button
+                className={styles['section-header']}
+                aria-expanded={open}
+                aria-controls={bodyId}
+                onClick={toggle}
+                style={headerStyle}
+            >
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)